.github/ISSUE_TEMPLATE/issue-report.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Issue report"
+description: Report documentation issues such as inaccuracies, broken links, typos, or missing information.
+title: "[Issue]: "
+labels: ["Issue"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi 👋. Thanks for taking the time to fill out this issue report!
+        This form will create an issue that Elastic's docs team will triage and prioritize.
+        You can also open a PR instead.
```

Comment:
>       Alternatively, you can create a pull request (link to contribution info)...

I don't think we have this sort of page yet, but would be nice to link to more info eventually.

Reply:
Could link here? https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/contribute/

Reply:
Let's leave this out for now. I don't think we've figured out how this link to docs-builder docs will or won't change in the future.